docs/concepts/terminology.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Terminology
+toc: true
+weight: 51
+indent: true
+---
+
+# Terminology
+
+## A Note on Style
+
+Each type of Kubernetes resource has a ‘Pascal case’ name - i.e. a title case
+name with no spaces between each word. Examples include ‘DaemonSet’ and
+‘PersistentVolumeClaim’. Often these names are written using fixed width fonts
+to draw attention to the fact that they’re a concrete type of resource within
+the API - e.g. `PersistentVolumeClaim`.
+
+Crossplane follows this convention. We often use names like RDSInstance or
+CompositeResourceDefinition when discussing Crossplane types. Crossplane also
+has “classes of types” - i.e. concepts that aren’t a distinct type of API
+resource, but rather describe a group of conceptually similar types. For example
+there is no ManagedResource type in Crossplane - instead types like RDSInstance
+and GKECluster are said to be “a managed resource”.
+
+Use your discretion as to whether you use pascal case when writing about a
+distinct type - e.g. “RDS Instance” and “RDSInstance” are both fine. The pascal
+case form makes more sense in contexts like documentation where you’re referring
+to Crossplane’s RDSInstance managed resource rather than the general concept of
+“an RDS instance”. Avoid using Pascal case when talking about classes of types -
+i.e. always write “managed resource”, not“ManagedResource”. Each of the below
+terms clarify whether they correspond to a single type, or a class of types.
+
+### Why 'X'?
+
+You may notice that Crossplane uses “X” as shorthand for “Crossplane” and/or
+“Composite”. This is because some of our concepts - specifically Composite
+Resources (XRs) and Composite Resource Definitions (XRDs) are modelled on
+similar Kubernetes concepts - Custom Resources (CRs) and Custom Resource
+Definitions (CRDs). We chose to abbreviate to (e.g.) XRD instead of CRD to avoid
+confusion.
+
+## Crossplane Terms
+
+The below terms are commonly used in the Crossplane ecosystem.
```

Comment:
This might read better.
```suggestion
The terms below are commonly used in the Crossplane ecosystem.
```

Reply:
Apparently this is a somewhat controversial point but it seems like both forms are valid so I'm going to be lazy and leave it. :) https://english.stackexchange.com/questions/74182/above-below-before-after-a-noun